bullet-sync/.env

```
@@ -0,0 +1,5 @@
+PORT=5175
+CAL_URL="https://server.smithery.ai/@goldk3y/google-calendar-mcp/mcp?api_key=cb280599-8d83-4558-b8ab-f4f8b53ba7d4&profile=swift-weasel-UBlSKN"
```

dotenv-linter (3.3.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)